client/src/common/translations.ts

```diff
@@ -1,24 +1,44 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
+import { MsOption } from '@/components/core/ms-types';
+import { InvitationStatus, WorkspaceRole } from '@/parsec';
 import { ComposerTranslation } from 'vue-i18n';
-import { WorkspaceRole, InvitationStatus } from '@/parsec';
 
-export function translateWorkspaceRole(t: ComposerTranslation, role: WorkspaceRole | null): string {
+export function translateWorkspaceRole(t: ComposerTranslation, role: WorkspaceRole | null): MsOption {
```

Comment:
This should not return a `MsOption`. Just define a dummy interface above, something like
```typescript
interface WorkspaceRoleTranslation {
  label: string;
  description?: string;
}
````
and use this as a return value. It doesn't need to be exported, it's never going to be used outside this function, and it's fine.